trust-store-trust-policy-specification.md

```diff
@@ -1,154 +1,88 @@
 
-### Artifact Integrity
-
-The artifact MUST be signed and has not been altered.
-
-### Artifact Expiry
-
-Trust policy allows users to define how the system should behave when the artifact's signature is expired or signing identity is expired or timestamping identity is expired.
-
-If artifact expiry validations are enforced the implementation MUST perform the following validations:
-
-1. If signature expiry is present then signature MUST NOT be expired.
-1. If signing identity is certificate and
-    1. The timestamp signature is not present then the signing certificate and the certificate chain MUST NOT be expired.
-    1. The timestamp signature is present then the timestamp signature MUST be valid.
-       At the time of timestamping, signing certificate (including certificate chain) MUST NOT be expired.
-       Also, the timestamping certificate and certificate chain MUST NOT be expired.
-
-### Artifact Revocation
-
-Trust policy also allows users to control how the system should behave when signing identity or  timestamping identity is revoked
-
-If revocation validations are enforced implementation MUST perform the following validations:
-
-1. If signing identity is a certificate then signing certificate and certificate chain MUST NOT be revoked.
-1. If the timestamp signature is present then the timestamping certificate and certificate chain MUST NOT be revoked.
-
-The implementation MUST support both [OCSP](https://datatracker.ietf.org/doc/html/rfc6960) and [CRL](https://datatracker.ietf.org/doc/html/rfc5280) based revocations.
-Since revocation check requires network call and network call can fail because of a variety of reasons such as revocation endpoint is unavailable, network connectivity issue, DDoS attack, etc the implementation MUST support both `fail-open` or `fail-close` use cases.
-
-- `fail-open`: If revocation endpoint is not reachable, consider artifact as not revoked.
-- `fail-close`: If revocation endpoint is not reachable, consider artifact as revoked.
-
 ### Trust Policy Schema
 
-The trust policy is represented as JSON data structure as shown below:
+The trust policy is as JSON document, example shown below:
 
 ```jsonc
 {
     "version": "1.0",
     "trustPolicies": [
         {
-            "name": "verify-signature",
-            "scopes": [
-                "registry.wabbit-networks.io/software/net-monitor"
-                "registry.wabbit-networks.io/software/net-logger" ],
-            "trustStores": [ "trust-store-name-1", "trust-store-name-2" ],
-            "trustAnchors": [
-                "subject: C=US, ST=WA, L=Seattle, O=acme-rockets.io"
+            "name": "policy-for-wabbit-networks-images",
+            "registryScopes": [
+                "registry.acme-rockets.io/software/net-monitor"
+                "registry.acme-rockets.io/software/net-logger" ],
+            "signatureVerification" : "strict",
+            "trustStore": "ca/wabbit-networks",
```

Comment:
I'm confused here since `ca` does not match `ca_roots`. Should we make everything consistent?

Reply:
Fixed the typo in latest version.